packages/compass-schema/src/components/type/type.jsx

```diff
@@ -107,39 +111,26 @@ class Type extends Component {
       width: percentage,
     };
     const subtypes = this._getArraySubTypes();
-    const label = (
-      <Disclaimer className="schema-field-type-label">
-        {this.props.name}
-      </Disclaimer>
-    );
+
     // show integer accuracy by default, but show one decimal point accuracy
     // when less than 1% or greater than 99% but no 0% or 100%
     const format =
       (this.props.probability > 0.99 && this.props.probability < 1.0) ||
       (this.props.probability > 0 && this.props.probability < 0.01)
         ? '0.0%'
         : '0%';
-    const tooltipText = `${this.props.name} (${numeral(
-      this.props.probability
-    ).format(format)})`;
-    const tooltipOptions = {
-      'data-for': schemaTooltipClass,
-      'data-tip': tooltipText,
-      'data-effect': 'solid',
-      'data-border': true,
-    };
-    tooltipOptions['data-offset'] = this.props.showSubTypes
-      ? '{"top": -25, "left": 0}'
-      : '{"top": 10, "left": 0}';
+    const labelText = `${this.props.name}${
+      this.props.probability !== 1
+        ? ` (${numeral(this.props.probability).format(format)})`
+        : ''
+    }`;
+    const label = (
+      <Disclaimer className={fieldTypeLabelStyles} title={labelText}>
+        {labelText}
+      </Disclaimer>
+    );
     return (
-      <button
-        {...tooltipOptions}
-        type="button"
-        className={cls}
-        style={style}
-        onClick={handleClick}
-      >
-        <ReactTooltip id={schemaTooltipClass} />
+      <button type="button" className={cls} style={style} onClick={handleClick}>
```

Comment:
Shouldn't we keep the tooltip here? is a bit different from the other buttons as is not a description for the icon, is more an actual content